app/controllers/admin/positions_controller.rb

```diff
@@ -0,0 +1,42 @@
+class Admin::PositionsController < ApplicationController
+  before_action -> { user_has_position(Position.first) }
```

Comment:
hmm. It just makes me nervous that somehow it is not the first position record somehow. If it isn't it completely breaks the system.